includes/admin/class-wcs-admin-meta-boxes.php

```diff
@@ -94,8 +95,7 @@ public function add_meta_boxes( $post_type = '', $post_or_order_object = null )
 		}
 
 		// Get "Edit Order" screen ID, which differs if HPOS is enabled.
-		$screen         = wcs_is_custom_order_tables_usage_enabled() ? wc_get_page_screen_id( 'shop-order' ) : 'shop_order';
-		$current_screen = get_current_screen();
```

Comment:
The `$current_screen` variable has inadvertently been removed here (I think). It is used by the if statement below.

Reply:
Will be resolved in #347